### PR TITLE
Information summary display for other

### LINKED ISSUE #1237: Personal Information Summary Block - display value for "other"
## Summary

on the confirmation page, the personal information doesn't use the value the user typed in for "other", it just uses the work "Other" taken from the checkboxes you checked.

## Design Detail

could use howItStarted page as reference.

### PR DESCRIPTION
**Pull request by kevinyang06**
Fixes #1237 

# Description

on the confirmation page, the personal information doesn't use the value the user typed in for "other", it just uses the word "Other" taken from the checkboxes you checked.

# Any new packages installed?

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code

# Required followup work

> encapsulate code into function for reusability

### CODE REVIEW

File: f2/src/summary/InformationSummary.js [hunk 1]
```diff
@@ -7,22 +7,70 @@ import { useStateValue } from '../utils/state'
...
-import { useIntl } from 'react-intl'
 import { useLingui } from '@lingui/react'
 import { Text } from '../components/text'
 
 export const InformationSummary = props => {
   const [data] = useStateValue()
   const { i18n } = useLingui()
-  const intl = useIntl()
+  const infoReqSummary = []
+  let infoReqLine = " "
```
Comment (sastels) [inline f2/src/summary/InformationSummary.js]: LGTM is complaining about this line assigning a value that isn't used - you can just do `let infoReqLine` instead (without the assignment)

File: f2/src/summary/InformationSummary.js [hunk 2]
```diff
@@ -7,22 +7,70 @@ import { useStateValue } from '../utils/state'
...
+  let infoReqSummaryLastItem = []
+  let infoReqSummaryFirstitems = []
+  const infoObtainedSummary = []
+  let infoObtainedLine = " "
```
Comment (sastels) [inline f2/src/summary/InformationSummary.js]: Similarly here `let infoObtainedLine`

File: f2/src/locales/fr.json [hunk 3]
```diff
@@ -248,7 +248,7 @@
   "typeOfInfoReq.homeAddress": "Adresse du domicile",
   "typeOfInfoReq.other": "Autre",
   "typeOfInfoReq.sin": "Numéro d'assurance sociale",
-  "whatHappenedPage.backButton": "Retourner à « Quelles pourraient-être les répercussions? »",
+  "whatHappenedPage.backButton": "Retourner à 'Quelles pourraient-être les répercussions?'",
```
[the remark this answers is not in the document]
Comment (kevinyang06) [reply f2/src/locales/fr.json]: fixed

File: f2/src/summary/InformationSummary.js (hunk 1 above)
Comment (kevinyang06) [reply f2/src/summary/InformationSummary.js]: fixed

File: f2/src/summary/InformationSummary.js (hunk 2 above)
Comment (kevinyang06) [reply f2/src/summary/InformationSummary.js]: fixed

File: f2/src/summary/InformationSummary.js [hunk 4]
```diff
@@ -62,20 +112,33 @@ export const InformationSummary = props => {
...
+              <Text>
+                <Trans id="confirmationPage.impactIntro" />
+              </Text>
+            )}
+          <DescriptionListItem
+            descriptionTitle="confirmationPage.personalInformation.tellUsMore"
+            description={personalInformation.tellUsMore}
+          />
+        </Stack>
+
```
Comment (sastels) [inline f2/src/summary/InformationSummary.js]: These lines are duplicated below and are causing your syntax error. (probably when you were fixing a merge conflict 😛 )

Comment (kevinyang06) [reply f2/src/summary/InformationSummary.js]: fixed

**Pull request merged**